lib/trino-orc/src/test/java/io/trino/orc/BenchmarkColumnReaders.java

```diff
@@ -997,6 +1039,100 @@ private Iterator<?> createValues()
         }
     }
 
+    @State(Thread)
+    public static class UUIDNoNullBenchmarkData
+            extends BenchmarkData
+    {
+        @Setup
+        public void setup()
+                throws Exception
+        {
+            setup(UUID, createValues());
+        }
+
+        private Iterator<?> createValues()
+        {
+            List<UUID> values = new ArrayList<>();
+            for (int i = 0; i < ROWS; ++i) {
+                values.add(java.util.UUID.randomUUID());
+            }
+            return values.iterator();
+        }
+    }
+
+    @State(Thread)
+    public static class UUIDWithNullBenchmarkData
+            extends BenchmarkData
+    {
+        @Setup
+        public void setup()
+                throws Exception
+        {
+            setup(UUID, createValues());
+        }
+
+        private Iterator<?> createValues()
+        {
+            List<UUID> values = new ArrayList<>();
+            for (int i = 0; i < ROWS; ++i) {
+                if (random.nextBoolean()) {
```

Comment:
50% null distribution is rare. It is usually sparse or dense (0.1, 0.9) Please add a `nullChance` param with values like 0, .1, .5, .9, 1?

Reply:
This is consistent with other benchmarks here.
This class would use a refactor but it's not in the scope of this PR.

Reply:
Up to you. If this is too much work, you can just leave it as it is

Reply:
Let's leave it as is. We can refactor this if we have more orc work.